clients/localdir/client.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package localdir is local repo containing source code.
+package localdir
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"go.uber.org/zap"
+
+	clients "github.com/ossf/scorecard/v2/clients"
+)
+
+var errUnsupportedFeature = errors.New("unsupported feature")
+
+type localDirClient struct {
+	logger *zap.Logger
+	ctx    context.Context
+}
+
+func (client *localDirClient) InitRepo(repo clients.Repo) error {
+	return nil
+}
+
+func (client *localDirClient) URL() string {
```

Comment:
Should we change the `RepoClient` API also to `URI`?

Reply:
good point, will do